ST_structs.h

```diff
@@ -1,333 +1,721 @@
+        recol[2];
 };
+
+/**
+ * \brief Ant mound information.
+ * 
+ * Used in the globals_st struct.
+ * 
+ * \sa globals_st
+ * 
+ * \ingroup MORTALITY
+ */
 struct antmounds_st {
+      /** \brief If TRUE ant mounds will be used. */ 
   Bool use;
+      /** \brief The probability of ant mound occurance. Between 0 and 1. */
   RealF occur;
+      /** \brief The minimum number of ant mounds to generate in a year. */
   IntUS minyr,
+      /** \brief The maximum number of ant mounds to generate in a year. */
        maxyr;
 };
+
+/**
+ * \brief Holds information on animal burrows.
+ * 
+ * Used in the globals_st struct.
+ * 
+ * \sa globals_st
+ * 
+ * \ingroup MORTALITY
+ */
 struct burrows_st {
+  /** \brief If TRUE burrows will be used. */
   Bool use;
+  /** \brief Value between 0 and 1. The probability that a burrow will occur. */
   RealF occur;
+  /** \brief Minimum number of burrows in a year. */
   IntUS minyr;
 };
 
+/**
+ * \brief Stores file names for output.
+ * 
+ * The struct contains the name of the yearly and summary output files.
+ * Used in the globals_st struct.
+ * 
+ * \sa globals_st
+ * 
+ * \ingroup STEPPE
+ */
 struct outfiles_st {
-  FILE *fp_year,  /* file handle for yearly so it can stay open*/
-       *fp_sumry; /* file handle for averages output */
-  IntUS suffixwidth; /* max width of outfile suffix if printing yearly */
+      /** \brief File handle for yearly output. */
+  FILE *fp_year,
+      /**\brief File handle for averages output. */
+       *fp_sumry;
+      /** \brief max width of outfile suffix if printing yearly. */
+  IntUS suffixwidth;
 };
 
+/**
+ * \brief Stores global information that is independent of any species or group.
+ * 
+ * globals_st deals mainly with structural variables like year and iteration. It also
+ * contains information on file names, precipitation, temperature, mounds, pats and burrows.
+ * This struct is instanciated by the global variable Globals.
+ * 
+ * \sa Globals
+ * 
+ * \ingroup STEPPE
+ */
 struct globals_st {
+  /** \brief Precipitation constants. \sa ppt_st */
   struct ppt_st ppt;
+  /** \brief Temperature constants. \sa temp_st */
   struct temp_st temp;
+  /** \brief Fecal pat constants. \sa fecalpats_st */
   struct fecalpats_st pat;
+  /** \brief Ant mound constants. \sa antmounds_st */
   struct antmounds_st mound;
+  /** \brief Animal burrow constants. \sa burrows_st */
   struct burrows_st burrow;
 
-  RealF plotsize,   /* size of plot in square meters */
-        gsppt_prop, /* proportion of ppt during growing season*/
-        tempparm[2][3]; /* three parms for Warm/Cool growth mod*/
-  IntUS runModelYears,  /* number of years to run the model*/
-      Max_Age,        /* oldest plant; same as runModelYears for now */
+      /** \brief Size of the plot in square meters */
+  RealF plotsize,
+      /** \brief Proportion of ppt during growing season. */
+        gsppt_prop,
+      /** \brief Three parameters for Warm/Cool growth modifiers. */
```

Comment:
change to:

warm-season/cool-season species growth modifiers

Reply:
Addressed by commit 984bd13.